### PR TITLE
pkg/trace/watchdog: fix CPU times retrieval

### PR DESCRIPTION
**Pull request by gbbr**
The previous code would not find the PID and would not report CPU times under many scenarios which was annoying. The new code uses the same module as our process agent which seems to be working much better.

### CODE REVIEW

File: pkg/trace/watchdog/cpu.go [hunk 1]
```diff
@@ -8,16 +8,19 @@
...
-	p, err := process.NewProcess(pid)
+	times, err := cpu.Times(false)
 	if err != nil {
 		return 0, err
 	}
-	times, err := p.Times()
-	if err != nil {
-		return 0, err
+	if len(times) == 0 {
+		return 0, errors.New("no CPU times returned. Will report 0 CPU usage.")
```
Comment (ajgajg1134) [inline pkg/trace/watchdog/cpu.go:23]: super-nit: capitalize the initial `no` since it's a full sentence

Comment (gbbr) [reply pkg/trace/watchdog/cpu.go:23]: Actually the idiom is to keep errors lowercased. The reason is that almost 99% of the time when logging an error you do something like:
```go
log.Error("Error doing something: %v", err)
```
... and then if the error is capitalised, it looks wrong.

**Review by ajgajg1134 — APPROVED** [review]

File: pkg/trace/watchdog/cpu.go (hunk 1 above)
Comment (gbbr) [reply pkg/trace/watchdog/cpu.go]: Yep, just noticed, thanks to your comment!

**Review by cswatt — APPROVED** [review]

**Comment by gbbr** [comment]
Closing this because the new code gets the CPU for the entire machine, not just the process.

**Pull request closed without merging**